<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit" />


<title>CD73 OverExpression in OV90 and OVCAR cell lines</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">UC_CD73_210707</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/FabioBedin/UC_CD73_210707">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">CD73 OverExpression in OV90 and OVCAR cell lines</h1>
<h4 class="author">Fabio Bedin | MS-Unit</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-08-10
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>UC_CD73_210707/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210707code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210707)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210707code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210707)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomFabioBedinUCCD73210707tree5e39369003122a6eb0fff2c176c9e403f61333a4targetblank5e39369a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/FabioBedin/UC_CD73_210707/tree/5e39369003122a6eb0fff2c176c9e403f61333a4" target="_blank">5e39369</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomFabioBedinUCCD73210707tree5e39369003122a6eb0fff2c176c9e403f61333a4targetblank5e39369a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/FabioBedin/UC_CD73_210707/tree/5e39369003122a6eb0fff2c176c9e403f61333a4" target="_blank">5e39369</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  analysis/OV90.Rmd
    Untracked:  analysis/OVCAR.Rmd
    Untracked:  code/custom_functions.R
    Untracked:  data/experimentalDesignTemplate.txt
    Untracked:  data/proteinGroups_UC_CD73.txt
    Untracked:  output/results_UC_CD73_210709.xlsx
    Untracked:  output/results_UC_CD73_210716.xlsx
    Untracked:  results_table.txt
    Untracked:  results_table.zip

Unstaged changes:
    Modified:   analysis/both_cell_line.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/both_cell_line.Rmd</code>) and HTML (<code>docs/both_cell_line.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/FabioBedin/UC_CD73_210707/5e39369003122a6eb0fff2c176c9e403f61333a4/docs/both_cell_line.html" target="_blank">5e39369</a>
</td>
<td>
FabioBedin
</td>
<td>
2021-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/FabioBedin/UC_CD73_210707/blob/5d45ce78775e4147d90590c1599739a6dba8b7ca/analysis/both_cell_line.Rmd" target="_blank">5d45ce7</a>
</td>
<td>
FabioBedin
</td>
<td>
2021-07-13
</td>
<td>
update GO treemap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/FabioBedin/UC_CD73_210707/5d45ce78775e4147d90590c1599739a6dba8b7ca/docs/both_cell_line.html" target="_blank">5d45ce7</a>
</td>
<td>
FabioBedin
</td>
<td>
2021-07-13
</td>
<td>
update GO treemap
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/FabioBedin/UC_CD73_210707/fd7e952ba14e95e0d43da17159bfe0d720d489bb/docs/both_cell_line.html" target="_blank">fd7e952</a>
</td>
<td>
FabioBedin
</td>
<td>
2021-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/FabioBedin/UC_CD73_210707/blob/7ac28575eb4670274dfb791475fa0c888ad0d172/analysis/both_cell_line.Rmd" target="_blank">7ac2857</a>
</td>
<td>
FabioBedin
</td>
<td>
2021-07-13
</td>
<td>
upload porject
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>knitr::opts_chunk$set(fig.align = &quot;center&quot;, warning=FALSE, message=FALSE, fig.height=8, fig.width=8, fig.retina = 3)
library(&quot;MBQN&quot;)
library(&quot;dplyr&quot;)
library(&quot;tidyr&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;tibble&quot;)
library(&quot;ggplot2&quot;)
library(&quot;enrichR&quot;)
library(&quot;DT&quot;)
library(&quot;stringr&quot;)
library(&quot;patchwork&quot;)
library(&quot;here&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;openxlsx&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)
select &lt;- dplyr::select</code></pre>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(&quot;data/proteinGroups_UC_CD73.txt&quot;), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)

data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= 2,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>In this analysis we compare <strong>OVCAR</strong> and <strong>OV90</strong> cell lines, with or without overexpression of <strong>CD73</strong>.</p>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/experimentalDesignTemplate.txt&quot;), header = T, stringsAsFactors = F)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

data_filt &lt;- filter_missval(data_se, thr = 1)</code></pre>
<p>Fist we generate some informative prols:</p>
<div id="proteins-per-samples" class="section level2">
<h2><strong>Proteins per samples</strong></h2>
<p>This plot show the number of proteins identify in each samples:</p>
<pre class="r"><code>plot_numbers_lables(data_se, b_size = 18, lab_size = 7)</code></pre>
<p><img src="figure/both_cell_line.Rmd/proteins-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 cell lines:</p>
<div id="ovcar" class="section level3">
<h3>OVCAR</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(&quot;OVCAR&quot;)) %&gt;% colnames()

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;OVCAR&quot;)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 8,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;coral2&quot;, &quot;turquoise3&quot;), each = 4),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/both_cell_line.Rmd/upset_1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="ov90" class="section level3">
<h3>OV90</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(&quot;OV90&quot;)) %&gt;% colnames()

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;OV90&quot;)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 8,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;coral2&quot;, &quot;turquoise3&quot;), each = 4),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/both_cell_line.Rmd/upset_2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-cell-lines" class="section level3">
<h3>Both cell lines</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 16,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;salmon1&quot;, &quot;red3&quot;, &quot;turquoise3&quot;, &quot;royalblue1&quot;), each = 4),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/both_cell_line.Rmd/upset_3-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="differential-enrichment-analysis" class="section level2 tabset tabset-fade">
<h2><strong>Differential enrichment analysis</strong></h2>
<p>Since we have 4 replicates per condition, we decided to filter the data for at least 3 out of 4 valid values in each condition.</p>
<pre class="r"><code>test &lt;- c(&quot;OV90_CD73_vs_OV90_WT&quot;, &quot;OVCAR_CD73_vs_OVCAR_WT&quot;)

data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = 0.05, lfc = 1)

results&lt;-get_results(dep)</code></pre>
<div id="pca-plot" class="section level3">
<h3>PCA plot</h3>
<pre class="r"><code>plot_pca(dep, x = 1, y = 2, n = 500, point_size = 4, indicate = &quot;condition&quot;)</code></pre>
<p><img src="figure/both_cell_line.Rmd/PCA_2-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-plot" class="section level3 tabset tabset-fade">
<h3>Volcano plot</h3>
<div id="ovcar-1" class="section level4">
<h4>OVCAR</h4>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/experimentalDesignTemplate.txt&quot;), header = T, stringsAsFactors = F)

conditions_CAR&lt;-c(&quot;OVCAR_WT&quot;, &quot;OVCAR_CD73&quot;)

expdesign &lt;- subset(expdesign, condition %in% conditions_CAR)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se_CAR &lt;- make_se(data_unique, columns, expdesign)

data_filt &lt;- filter_by_grouping(data_se_CAR, thr = 1)

test_CAR &lt;- &quot;OVCAR_CD73_vs_OVCAR_WT&quot;

data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test_CAR)

dep &lt;- add_rejections(data_diff, alpha = 0.05, lfc = 1)

results_CAR&lt;-get_results(dep)

plot_volcano(dep, contrast=test_CAR, add_names=T,label_size=5,adjusted = F)</code></pre>
<p><img src="figure/both_cell_line.Rmd/ovcar_analysis-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>condz &lt;- &quot;OVCAR&quot;

results_DEP_OVCAR &lt;- get_df_wide(dep) %&gt;% 
  select(c(name, Protein.IDs, Protein.names, starts_with(condz), -ends_with(c(&quot;CI.R&quot;, &quot;CI.L&quot;)))) %&gt;% 
  dplyr::rename_with(., ~ gsub(pattern = paste0(test_CAR, &quot;_&quot;), replacement = &quot;&quot;, .), starts_with(test_CAR)) %&gt;% 
  dplyr::rename_with(., ~ gsub(pattern = condz , replacement = &quot;LFQ_intensity&quot;, .), starts_with(condz)) %&gt;% 
  dplyr::rename(FC = diff) %&gt;% 
  mutate(significant = if_else(significant, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  mutate(across(c(p.adj, p.val), format, scientific = T, digits = 2)) %&gt;% 
  mutate(across(starts_with(c(&quot;LFQ&quot;, &quot;FC&quot;)), .fns = ~ round(.x, digits = 2))) %&gt;% 
  relocate(significant) %&gt;%
  relocate(starts_with(&quot;LFQ&quot;), .after = p.val) %&gt;% 
  arrange(desc(significant))</code></pre>
</div>
<div id="ov90-1" class="section level4">
<h4>OV90</h4>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/experimentalDesignTemplate.txt&quot;), header = T, stringsAsFactors = F)

conditions_90&lt;-c(&quot;OV90_WT&quot;, &quot;OV90_CD73&quot;)

expdesign &lt;- subset(expdesign, condition %in% conditions_90)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se_90 &lt;- make_se(data_unique, columns, expdesign)

data_filt &lt;- filter_by_grouping(data_se_90, thr = 1)

test_90 &lt;- &quot;OV90_CD73_vs_OV90_WT&quot;

data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test_90)

dep &lt;- add_rejections(data_diff, alpha = 0.05, lfc = 1)

results_90&lt;-get_results(dep)

plot_volcano(dep, contrast=test_90, add_names=T,label_size=5,adjusted = F)</code></pre>
<p><img src="figure/both_cell_line.Rmd/ov90_analysis-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code>condz &lt;- &quot;OV90&quot;

results_DEP_OV90 &lt;- get_df_wide(dep) %&gt;% 
  select(c(name, Protein.IDs, Protein.names, starts_with(condz), -ends_with(c(&quot;CI.R&quot;, &quot;CI.L&quot;)))) %&gt;% 
  dplyr::rename_with(., ~ gsub(pattern = paste0(test_90, &quot;_&quot;), replacement = &quot;&quot;, .), starts_with(test_90)) %&gt;% 
  dplyr::rename_with(., ~ gsub(pattern = condz , replacement = &quot;LFQ_intensity&quot;, .), starts_with(condz)) %&gt;% 
  dplyr::rename(FC = diff) %&gt;% 
  mutate(significant = if_else(significant, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  mutate(across(c(p.adj, p.val), format, scientific = T, digits = 2)) %&gt;% 
  mutate(across(starts_with(c(&quot;LFQ&quot;, &quot;FC&quot;)), .fns = ~ round(.x, digits = 2))) %&gt;% 
  relocate(significant) %&gt;%
  relocate(starts_with(&quot;LFQ&quot;), .after = p.val) %&gt;% 
  arrange(desc(significant))</code></pre>
</div>
</div>
<div id="results-table" class="section level3 tabset tabset-fade">
<h3>Results table</h3>
<p>In this interactive table are summarized the restults:</p>
<div id="ovcar-2" class="section level4">
<h4>OVCAR</h4>
<pre class="r"><code>results_CAR %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test_CAR)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test_CAR,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test_CAR)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(FC = ratio) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;FC&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-8531ab0ac6cc692db9ae" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8531ab0ac6cc692db9ae">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123"],["CP","PTGS1","CD70","CLU","MYEF2","RAB25","ENDOD1","PARP9","PYGL","SSU72","ACSS3","ANPEP","C3","GSTM3","S100A9","MAOB","COL18A1","DTX3L","CD109","CPA4","C4orf27","GLDC","GALNT3","TNFAIP2","PVRL3","CRABP2","FAM49B","MFN1","GLS","H1FX","HKDC1","ALDH5A1","H1F0","PKP2","S100A1","IDH2","HIST1H1C","SLC25A1","TPM1","ALCAM","LRRC8D","NCAM1","TMCC3","SEPT8","NUSAP1","SERPINB5","EPS8L2","EXD2","CD44","GAA","GNAQ","PRCP","TMX4","GLS.1","PML","H2AFY2","CENPF","HGH1","SCEL","SIL1","CD82","LRCH2","CGREF1","NDNL2","RPE","ASS1","ITPR2","CDCA8","PUSL1","CD63","HID1","NF1","FADS1","APOL2","IQGAP2","EPS8","FSCN1","GSTM1","DHODH","RAB6B","ABCD1","BLMH","SIGIRR","CYP4F11","S100A8","MAOA","ZNF185.1","INCENP","SLC7A5","PNP","CLIP1","DHX29","DSC2","ITGB4","EEFSEC","SH2D4A","TPM2","SEPT10","UGT8","LAMB2","CRYAB","C6orf132","CIB1","GCAT","LDAH","MECOM","GSTM2","PARP14","ACSS2","CDC123","FAM49A","CFL2","PI4KA","SRGAP3","PPOX","TMEM128","APOE","MUC16","ZNF185","MVD","RAP1GDS1","SCNN1A","TACC1"],["9.4e-08","1.7e-07","3.5e-07","1.1e-06","1.1e-06","1.2e-06","1.9e-06","2.1e-06","1.8e-06","1.9e-06","3.0e-06","3.2e-06","2.7e-06","3.4e-06","3.0e-06","4.7e-06","5.3e-06","5.7e-06","8.5e-06","8.1e-06","9.2e-06","1.1e-05","1.4e-05","1.4e-05","1.5e-05","1.6e-05","1.6e-05","1.6e-05","2.0e-05","2.0e-05","2.0e-05","2.5e-05","2.8e-05","2.8e-05","3.0e-05","3.1e-05","3.4e-05","3.4e-05","3.5e-05","3.7e-05","3.9e-05","3.7e-05","3.9e-05","4.8e-05","5.1e-05","5.4e-05","5.8e-05","5.8e-05","6.5e-05","6.3e-05","6.2e-05","6.4e-05","6.5e-05","8.5e-05","1.1e-04","1.1e-04","1.3e-04","1.4e-04","1.3e-04","1.4e-04","1.5e-04","1.7e-04","1.7e-04","2.0e-04","2.1e-04","2.2e-04","2.3e-04","2.4e-04","2.5e-04","2.5e-04","2.6e-04","2.8e-04","3.2e-04","3.4e-04","3.4e-04","3.6e-04","4.0e-04","4.2e-04","4.3e-04","4.4e-04","4.6e-04","4.8e-04","4.8e-04","5.3e-04","5.3e-04","5.5e-04","5.4e-04","5.9e-04","5.8e-04","7.2e-04","7.9e-04","8.0e-04","8.3e-04","8.3e-04","9.0e-04","8.8e-04","9.7e-04","9.9e-04","1.2e-03","1.3e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.9e-03","2.0e-03","2.3e-03","2.5e-03","2.5e-03","2.6e-03","2.6e-03","2.8e-03","2.9e-03","3.1e-03","3.1e-03"],["3.7e-04","3.7e-04","5.1e-04","8.4e-04","8.4e-04","8.4e-04","9.0e-04","9.0e-04","9.0e-04","9.0e-04","9.9e-04","9.9e-04","9.9e-04","9.9e-04","9.9e-04","1.3e-03","1.4e-03","1.4e-03","1.8e-03","1.8e-03","1.8e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.5e-03","2.5e-03","2.5e-03","3.0e-03","3.2e-03","3.2e-03","3.3e-03","3.4e-03","3.5e-03","3.5e-03","3.5e-03","3.6e-03","3.6e-03","3.6e-03","3.6e-03","4.2e-03","4.4e-03","4.6e-03","4.7e-03","4.7e-03","4.8e-03","4.8e-03","4.8e-03","4.8e-03","4.8e-03","5.8e-03","6.9e-03","6.9e-03","7.8e-03","7.8e-03","7.8e-03","7.8e-03","8.2e-03","9.0e-03","9.1e-03","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","2.2e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.6e-02","2.6e-02","2.9e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.5e-02","3.8e-02","4.0e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.5e-02","4.7e-02","4.7e-02","5.0e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[2.99,4.33,-4.85,1.7,1.87,1.49,-1.41,1.85,-4.62,2,1.12,1.6,2.29,-2.17,-2.59,3.71,1.37,1.78,-3.14,-2.64,-1.55,1.38,1.01,1.42,1.93,-2.02,-1.07,1.45,-1.37,1.27,3.44,1.46,1.05,1.06,1.98,1.17,2.27,1.27,-1.22,1.26,3.17,1.44,1.6,1.02,1.06,-1.69,1.18,-1.26,-1.49,1.16,1,-1.13,1.52,-1.36,1.18,1.37,1.49,-3.49,1.81,1.27,1.44,-2.96,1.37,-1.02,-1.02,1.13,1.64,1.45,1.23,1.68,1.99,-3.42,1.31,1.18,-1.6,2.46,-2.87,-2,1.09,1.38,1.6,-1.06,1.08,-2.69,-2.98,1.79,1.16,1.72,-1.11,-1.36,-1.28,-1.39,1.34,3.63,1.36,-1.01,-1.19,1.08,2.27,1.09,-3.74,1.5,1.25,-1.04,1.12,1.62,-1.62,1.3,-2.14,-1.12,-1.25,-1.53,-1.21,1.1,1.05,1.01,-1.58,1.26,1.83,-1.25,-1.71,1.44,-1.18]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>FC<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="ov90-2" class="section level4">
<h4>OV90</h4>
<pre class="r"><code>results_90 %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test_90)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test_90,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test_90)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(FC = ratio) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;FC&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-2f4d7b5320edbc94a7b0" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2f4d7b5320edbc94a7b0">{"x":{"filter":"none","data":[["1","2","3","4","5","6"],["NT5E","CLCA1","HHIP","ITPR1","PECAM1","ANPEP"],["8.4e-09","4.3e-08","3.9e-06","3.7e-06","9.9e-06","2.1e-05"],["3.5e-05","9.0e-05","4.0e-03","4.0e-03","8.1e-03","1.5e-02"],["True","True","True","True","True","True"],[12.1,-3,-1.31,1.56,-6.2,-2.89]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>FC<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="common-dep" class="section level4">
<h4>Common DEP</h4>
<pre class="r"><code>results &lt;- merge(results_CAR, results_90, by.x=&quot;ID&quot;, by.y=&quot;ID&quot;, all.x = TRUE, all.y = TRUE)

results %&gt;% 
  mutate(name = case_when(is.na(name.x) ~ name.y, TRUE ~ name.x)) %&gt;% 
  mutate(significant.x = case_when(is.na(significant.x) ~ FALSE, TRUE ~ significant.x)) %&gt;% 
  mutate(significant.y = case_when(is.na(significant.y) ~ FALSE, TRUE ~ significant.y)) %&gt;% 
  mutate(significant = if_else(significant.x &amp; significant.y, TRUE, FALSE)) %&gt;% 
  filter(significant) %&gt;%
  select(-significant, -ends_with(&quot;centered&quot;), -contains(c(&quot;.x&quot;, &quot;.y&quot;))) %&gt;% 
  pivot_longer(
   cols = ends_with(c(&quot;p.val&quot;, &quot;p.adj&quot;, &quot;significant&quot;, &quot;ratio&quot;)),
   names_to = c(&quot;comparison&quot;, &quot;.value&quot;),
   names_pattern = &quot;(.*_.*_.*_.*_.*)_(.*)&quot;) %&gt;% 
  arrange(name, p.adj) %&gt;% 
  relocate(comparison, name) %&gt;% 
  mutate(across(ends_with(c(&quot;p.val&quot;, &quot;p.adj&quot;)), format, scientific = T, digits = 2)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(FC = ratio) %&gt;% 
  drop_na() %&gt;%
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:7)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;FC&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-9eb3d8e05022e4e35f19" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9eb3d8e05022e4e35f19">{"x":{"filter":"none","data":[["1","2"],["OVCAR_CD73_vs_OVCAR_WT","OV90_CD73_vs_OV90_WT"],["ANPEP","ANPEP"],["P15144","P15144"],["3.2e-06","2.1e-05"],["9.9e-04","1.5e-02"],["True","True"],[1.6,-2.89]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>comparison<\/th>\n      <th>name<\/th>\n      <th>ID<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>FC<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6,7]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[7]; $(this.api().cell(row, 7).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[6]; $(this.api().cell(row, 6).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="unique-proteins" class="section level2 tabset tabset-fade">
<h2><strong>Unique Proteins</strong></h2>
<div id="ovcar-3" class="section level3">
<h3>OVCAR</h3>
<pre class="r"><code>data_filt_CAR &lt;- filter_missval(data_se_CAR, thr = 1)

unique_1 &lt;- assay(data_filt_CAR) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(conditions_CAR[1])) %&gt;%
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = case_when(is.na(intensity) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(rowname) %&gt;% 
  mutate(unique = if_else(sum(intensity) == 0, T, F)) %&gt;% 
  ungroup() %&gt;% 
  filter(unique) %&gt;% 
  pull(rowname) %&gt;%
  unique()
  
unique_2 &lt;- assay(data_filt_CAR) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(conditions_CAR[2])) %&gt;%
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = case_when(is.na(intensity) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(rowname) %&gt;% 
  mutate(unique = if_else(sum(intensity) == 0, T, F)) %&gt;% 
  ungroup() %&gt;% 
  filter(unique) %&gt;% 
  pull(rowname) %&gt;%
  unique()

table_unique_CAR &lt;- assay(data_filt_CAR) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  mutate(!!paste0(&quot;unique_&quot;, conditions_CAR[2]) := rowname %in% unique_1) %&gt;% 
  mutate(!!paste0(&quot;unique_&quot;, conditions_CAR[1]) := rowname %in% unique_2) %&gt;% 
  filter(get(paste0(&quot;unique_&quot;, conditions_CAR[1])) | get(paste0(&quot;unique_&quot;, conditions_CAR[2]))) %&gt;% 
  relocate(where(is.logical), .after = where(is.character)) %&gt;% 
  dplyr::rename(name = rowname)

table_unique_CAR %&gt;% 
  select(1:3) %&gt;%
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:3)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;unique_OVCAR_CD73&#39;, color = styleEqual(c(TRUE, FALSE), c(&#39;green&#39;, &#39;red&#39;))) %&gt;% 
  formatStyle(&#39;unique_OVCAR_WT&#39;, color = styleEqual(c(TRUE, FALSE), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-ac130c08b452fe6647f9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-ac130c08b452fe6647f9">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52"],["ANTXR2","AP1G2","ARL6IP6","BCAT1","BCL10","BIN1","C1GALT1C1","C4orf32","CAV2","CCPG1","CDK13","CEBPG","CELSR1","CEND1","DNLZ","DNTTIP1","DPP4","DUS2","ERCC6","FER1L6","GAN","GUK1","HSP90AB2P","IQSEC2","KANSL1","KMT2C","LCN2","LIMS1","LRRC8E","MAFK","MMTAG2","NDFIP2","NMES1","NUDT6","P4HTM","PDZK1","PLCB4","PRKAR2B","PTK2","PYCRL","RAB27B","RASAL2","RHOF","RINT1","SCAMP2","SERPINB7","STARD3","SYK","TPM3.1","UBE2T","WASH3P","ZRSR2"],[true,true,false,true,true,false,true,true,false,true,true,false,true,true,true,false,false,false,false,false,false,true,false,false,true,false,true,false,true,true,true,false,false,true,true,false,true,false,false,false,false,true,true,true,true,true,false,true,false,true,false,true],[false,false,true,false,false,true,false,false,true,false,false,true,false,false,false,true,true,true,true,true,true,false,true,true,false,true,false,true,false,false,false,true,true,false,false,true,false,true,true,true,true,false,false,false,false,false,true,false,true,false,true,false]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>unique_OVCAR_CD73<\/th>\n      <th>unique_OVCAR_WT<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<pre class="r"><code>table_unique_CAR_2 &lt;- table_unique_CAR %&gt;% 
  pivot_longer(!c(name, unique_OVCAR_CD73, unique_OVCAR_WT), names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(cond = gsub(pattern = &quot;..$&quot;, &quot;&quot;, samples)) %&gt;% 
  mutate(intensity = case_when(is.na(intensity) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(name, cond) %&gt;% 
  mutate(Unique_3_out_of_4 = case_when(sum(intensity) == 3 ~ &quot;+&quot;)) %&gt;% 
  mutate(Unique_4_out_of_4 = case_when(sum(intensity) == 4 ~ &quot;+&quot;)) %&gt;% 
  ungroup() %&gt;% 
  select(-cond) %&gt;% 
  group_by(name) %&gt;% 
  fill(Unique_3_out_of_4, .direction = &quot;downup&quot;) %&gt;% 
  fill(Unique_4_out_of_4, .direction = &quot;downup&quot;) %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  select(name, Unique_3_out_of_4, Unique_4_out_of_4) %&gt;% 
  mutate(across(starts_with(&quot;Unique_&quot;), ~ case_when(is.na(.x) ~ &quot;&quot;, TRUE ~ as.character(.x))))

condz &lt;- &quot;OVCAR&quot;

result_unique_OVCAR &lt;- get_df_wide(data_filt_CAR) %&gt;% 
  select(c(name, Protein.IDs, Protein.names)) %&gt;% 
  right_join(table_unique_CAR) %&gt;% 
  left_join(table_unique_CAR_2) %&gt;% 
  dplyr::rename_with(., ~ gsub(pattern = condz , replacement = &quot;LFQ_intensity&quot;, .), starts_with(condz)) %&gt;% 
  mutate(unique_OVCAR_CD73 = if_else(unique_OVCAR_CD73, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  mutate(unique_OVCAR_WT = if_else(unique_OVCAR_WT, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  mutate(across(starts_with(&quot;LFQ&quot;), .fns = ~ round(.x, digits = 2)))</code></pre>
</div>
<div id="ov90-3" class="section level3">
<h3>OV90</h3>
<pre class="r"><code>data_filt_90 &lt;- filter_missval(data_se_90, thr = 1)

unique_1 &lt;- assay(data_filt_90) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(conditions_90[1])) %&gt;%
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = case_when(is.na(intensity) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(rowname) %&gt;% 
  mutate(unique = if_else(sum(intensity) == 0, T, F)) %&gt;% 
  ungroup() %&gt;% 
  filter(unique) %&gt;% 
  pull(rowname) %&gt;%
  unique()
  
unique_2 &lt;- assay(data_filt_90) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(conditions_90[2])) %&gt;%
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = case_when(is.na(intensity) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(rowname) %&gt;% 
  mutate(unique = if_else(sum(intensity) == 0, T, F)) %&gt;% 
  ungroup() %&gt;% 
  filter(unique) %&gt;% 
  pull(rowname) %&gt;%
  unique()

table_unique_90 &lt;- assay(data_filt_90) %&gt;%
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  mutate(!!paste0(&quot;unique_&quot;, conditions_90[2]) := rowname %in% unique_1) %&gt;% 
  mutate(!!paste0(&quot;unique_&quot;, conditions_90[1]) := rowname %in% unique_2) %&gt;% 
  filter(get(paste0(&quot;unique_&quot;, conditions_90[1])) | get(paste0(&quot;unique_&quot;, conditions_90[2]))) %&gt;% 
  relocate(where(is.logical), .after = where(is.character)) %&gt;% 
  dplyr::rename(name = rowname) 

table_unique_90_2 &lt;- table_unique_90 %&gt;% 
  pivot_longer(!c(name, unique_OV90_CD73, unique_OV90_WT), names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(cond = gsub(pattern = &quot;..$&quot;, &quot;&quot;, samples)) %&gt;% 
  mutate(intensity = case_when(is.na(intensity) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(name, cond) %&gt;% 
  mutate(Unique_3_out_of_4 = if_else(sum(intensity) == 3, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  mutate(Unique_4_out_of_4 = if_else(sum(intensity) == 4, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  ungroup() %&gt;% 
  select(-cond) %&gt;% 
  group_by(name) %&gt;% 
  fill(Unique_3_out_of_4, .direction = &quot;downup&quot;) %&gt;% 
  fill(Unique_4_out_of_4, .direction = &quot;downup&quot;) %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  select(name, Unique_3_out_of_4, Unique_4_out_of_4)
  
table_unique_90 %&gt;% 
  select(1:3) %&gt;%
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:3)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;unique_OV90_CD73&#39;, color = styleEqual(c(TRUE, FALSE), c(&#39;green&#39;, &#39;red&#39;))) %&gt;% 
  formatStyle(&#39;unique_OV90_WT&#39;, color = styleEqual(c(TRUE, FALSE), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-88a3f1d7868ba3821b3d" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-88a3f1d7868ba3821b3d">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34"],["ALG10B","ANKS1A","AP1M2","CBLL1","CCNB2V","CDKN2AIPNL","COA3","DENND4C","DNTTIP1","FNTB","FOSL2","HS1BP3","HSPA4L","MUC16","NDUFB4","NLE1","NOS3","PCSK6","PHF8","PPOX","RAB36","RGPD5","RNF31","SERF2","SNX12","TAP2","TDRD3","TEFM","THEM4","TTC7A","USP8","VEZT","VPS37C","WDR35"],[false,true,true,true,false,false,true,true,false,true,false,false,false,false,true,false,false,true,true,false,false,false,false,false,true,true,false,false,false,false,true,false,true,false],[true,false,false,false,true,true,false,false,true,false,true,true,true,true,false,true,true,false,false,true,true,true,true,true,false,false,true,true,true,true,false,true,false,true]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>unique_OV90_CD73<\/th>\n      <th>unique_OV90_WT<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<pre class="r"><code>condz &lt;- &quot;OV90&quot;

result_unique_OV90 &lt;- get_df_wide(data_filt_90) %&gt;%
  select(c(name, Protein.IDs, Protein.names)) %&gt;% 
  right_join(table_unique_90) %&gt;% 
  left_join(table_unique_90_2) %&gt;% 
  dplyr::rename_with(., ~ gsub(pattern = condz , replacement = &quot;LFQ_intensity&quot;, .), starts_with(condz)) %&gt;% 
  mutate(unique_OV90_CD73 = if_else(unique_OV90_CD73, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  mutate(unique_OV90_WT = if_else(unique_OV90_WT, &quot;+&quot;, &quot;&quot;)) %&gt;% 
  mutate(across(starts_with(&quot;LFQ&quot;), .fns = ~ round(.x, digits = 2)))</code></pre>
</div>
</div>
<div id="gene-ontology" class="section level2 tabset tabset-fade">
<h2><strong>Gene Ontology</strong></h2>
<p>Then we perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a KEGG ontology.</p>
<p>The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong>:</p>
<div id="ovcar-4" class="section level3 tabset tabset-fade">
<h3>OVCAR</h3>
<div id="cd73" class="section level4">
<h4>CD73</h4>
<pre class="r"><code>Gname &lt;- results_CAR %&gt;% filter(significant &amp; get(paste0(test_CAR, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

Uni &lt;- table_unique_CAR %&gt;% filter(unique_OVCAR_CD73) %&gt;% pull(name) %&gt;% unique()

#bg_list &lt;- results_CAR %&gt;% pull(name) %&gt;% unique()

GO_name &lt;- c(Gname, Uni)

GO &lt;- gost(query = GO_name, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T)

gostplot(GO, capped = TRUE, interactive = TRUE)</code></pre>
<div id="htmlwidget-5253cdbcdd12e45c3736" style="width:768px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-5253cdbcdd12e45c3736">{"x":{"data":[{"x":[120.345729257727,140.898322533677,132.442376145034,79.7244085813207,171.998411466526],"y":[2.45566374789867,1.76415362776014,1.58924068314059,1.56480365629461,1.40383121498338],"text":["GO:0042135 (11) <br> neurotransmitter catabolic process <br> 3.502e-03","GO:0055114 (475) <br> obsolete oxidation-reduction process <br> 1.721e-02","GO:0048260 (53) <br> positive regulation of receptor-mediated endocytosis <br> 2.575e-02","GO:0002526 (115) <br> acute inflammatory response <br> 2.724e-02","GO:1901568 (81) <br> fatty acid derivative metabolic process <br> 3.946e-02"],"key":["GO:0042135","GO:0055114","GO:0048260","GO:0002526","GO:1901568"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[13.2227649307199,18.9190778923486,15.9306674399917,17.0632809089775,16.5632703112881],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData8531ab0a","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[64.5244030636107,69.6488993741428,62.1358335723345,64.4307336717959,59.4896732535677,54.7593689669227,57.0074343704768,62.1241248983576,54.9037759459705,55.3213853178112,58.5685909007227,55.415054709626,54.0802658762658,55.4189576009516,68.0799370612457,55.4462778402309,55.4501807315565,62.120222007032,62.1397364636601,55.3252882091369,55.4853067534871,59.5794397540569,67.7989288858014,59.4389356663347,59.466255905614,54.9271932939242,59.4350327750091],"y":[7.94580200774095,7.62838929321048,7.62108286833317,7.62108286833317,4.50359158169072,4.26885719650351,4.26011496726029,4.23092925158416,3.93525270070853,3.89343956651875,3.58394608735677,3.57088073928997,3.57088073928997,3.2538030862933,3.2538030862933,3.12616391483573,2.87921552977105,2.85162582159008,2.52005048507414,2.45098883865762,2.19107047904706,1.81458168669526,1.67107322137928,1.60381249274944,1.60381249274944,1.5406679205817,1.31522391451922],"text":["GO:0070062 (2178) <br> extracellular exosome <br> 1.133e-08","GO:1903561 (2263) <br> extracellular vesicle <br> 2.353e-08","GO:0043230 (2265) <br> extracellular organelle <br> 2.393e-08","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 2.393e-08","GO:0031982 (4058) <br> vesicle <br> 3.136e-05","GO:0005576 (4564) <br> extracellular region <br> 5.384e-05","GO:0012505 (4651) <br> endomembrane system <br> 5.494e-05","GO:0043227 (13201) <br> membrane-bounded organelle <br> 5.876e-05","GO:0005615 (3595) <br> extracellular space <br> 1.161e-04","GO:0005737 (11951) <br> cytoplasm <br> 1.278e-04","GO:0031090 (3644) <br> organelle membrane <br> 2.606e-04","GO:0005764 (723) <br> lysosome <br> 2.686e-04","GO:0000323 (723) <br> lytic vacuole <br> 2.686e-04","GO:0005765 (395) <br> lysosomal membrane <br> 5.574e-04","GO:0098852 (395) <br> lytic vacuole membrane <br> 5.574e-04","GO:0005773 (818) <br> vacuole <br> 7.479e-04","GO:0005774 (450) <br> vacuolar membrane <br> 1.321e-03","GO:0043226 (14027) <br> organelle <br> 1.407e-03","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 3.020e-03","GO:0005739 (1678) <br> mitochondrion <br> 3.540e-03","GO:0005783 (1995) <br> endoplasmic reticulum <br> 6.441e-03","GO:0032133 (6) <br> chromosome passenger complex <br> 1.533e-02","GO:0098588 (2165) <br> bounding membrane of organelle <br> 2.133e-02","GO:0031967 (1244) <br> organelle envelope <br> 2.490e-02","GO:0031975 (1244) <br> envelope <br> 2.490e-02","GO:0005622 (14808) <br> intracellular anatomical structure <br> 2.880e-02","GO:0031966 (750) <br> mitochondrial membrane <br> 4.839e-02"],"key":["GO:0070062","GO:1903561","GO:0043230","GO:0065010","GO:0031982","GO:0005576","GO:0012505","GO:0043227","GO:0005615","GO:0005737","GO:0031090","GO:0005764","GO:0000323","GO:0005765","GO:0098852","GO:0005773","GO:0005774","GO:0043226","GO:0043231","GO:0005739","GO:0005783","GO:0032133","GO:0098588","GO:0031967","GO:0031975","GO:0005622","GO:0031966"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.6862949001828,20.7283484272855,20.7293175601055,20.7293175601055,21.3573749246266,21.4812432421879,21.5010671808456,22.5637937602723,21.2287521888284,22.4650622831716,21.2431726843939,19.4265327462351,19.4265327462351,18.6908420061388,18.6908420061388,19.5725588879439,18.8525263258535,22.6237759212874,22.4729396933297,20.396981973735,20.5894944378604,11.9424497149374,20.6797093746665,20.0586642271833,20.0586642271833,22.6771653543307,19.470042500928],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData8531ab0a","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[16.1342530068331,42.4242757862463,15.8727966425433,15.0415995441293,41.7725860424194,9.75783958519719],"y":[2.14996841033511,2.03884460793489,1.47032573551542,1.4588743429385,1.34327243281903,1.34168910629721],"text":["GO:0016638 (20) <br> oxidoreductase activity, acting on the CH-NH2 group of donors <br> 7.080e-03","GO:0097621 (3) <br> monoamine oxidase activity <br> 9.144e-03","GO:0016491 (770) <br> oxidoreductase activity <br> 3.386e-02","GO:0015607 (1) <br> ABC-type fatty-acyl-CoA transporter activity <br> 3.476e-02","GO:0086083 (5) <br> cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication <br> 4.537e-02","GO:0008131 (6) <br> primary amine oxidase activity <br> 4.553e-02"],"key":["GO:0016638","GO:0097621","GO:0016491","GO:0015607","GO:0086083","GO:0008131"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[14.3345047676709,10.1713989351716,19.5011994481152,3.77952755905512,11.5159987834602,11.9424497149374],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData8531ab0a","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[197.971130649918],"y":[1.46126104832198],"text":"KEGG:00250 (37) <br> Alanine, aspartate and glutamate metabolism <br> 3.457e-02","key":["KEGG:00250"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(221,68,119,1)","opacity":0.8,"size":15.3676909120491,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(221,68,119,1)"}},"hoveron":"points","set":"SharedData8531ab0a","name":"KEGG","legendgroup":"KEGG","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[2,49.9206590071537],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8226750487752,71.2217645783655],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.123780619987,193.971385215695],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.873401257316,200.003902016042],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":45.251350032552,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9603295035768,62.5222198135703,134.547582917841,198.938651636679],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.00905891102808911,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"1ed439e344bf":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"1ed46bee7542":{"x":{},"xend":{},"y":{},"yend":{}},"1ed4344f61fc":{"x":{},"xend":{},"y":{},"yend":{}},"1ed4361617cf":{"x":{},"xend":{},"y":{},"yend":{}},"1ed433ed385b":{"x":{},"xend":{},"y":{},"yend":{}},"1ed465772d08":{"x":{},"y":{}},"1ed43cd15b59":{"yintercept":{}}},"cur_data":"1ed439e344bf","visdat":{"1ed439e344bf":["function (y) ","x"],"1ed46bee7542":["function (y) ","x"],"1ed4344f61fc":["function (y) ","x"],"1ed4361617cf":["function (y) ","x"],"1ed433ed385b":["function (y) ","x"],"1ed465772d08":["function (y) ","x"],"1ed43cd15b59":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData8531ab0a"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-cd73" class="section level4">
<h4>Table CD73</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>GO$result %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
  arrange(p_value) %&gt;%  
  mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))</code></pre>
<div id="htmlwidget-0df18093077b0751a457" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0df18093077b0751a457">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:MF","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","KEGG","GO:MF","GO:BP","GO:MF","GO:MF","GO:CC"],[2743,4056,2131,2719,1453,241,817,2128,278,385,1217,409,67,410,3654,417,418,2127,2132,11590,386,427,3623,10360,1476,16857,3582,1440,1447,14690,1180,284,3556,26,3343,24827,10193,1989,1439],["extracellular exosome","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","vesicle","extracellular region","endomembrane system","membrane-bounded organelle","extracellular space","cytoplasm","organelle membrane","lysosome","lytic vacuole","lysosomal membrane","lytic vacuole membrane","vacuole","vacuolar membrane","organelle","intracellular membrane-bounded organelle","neurotransmitter catabolic process","mitochondrion","endoplasmic reticulum","oxidoreductase activity, acting on the CH-NH2 group of donors","monoamine oxidase activity","chromosome passenger complex","obsolete oxidation-reduction process","bounding membrane of organelle","organelle envelope","envelope","positive regulation of receptor-mediated endocytosis","acute inflammatory response","intracellular anatomical structure","oxidoreductase activity","Alanine, aspartate and glutamate metabolism","ABC-type fatty-acyl-CoA transporter activity","fatty acid derivative metabolic process","cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication","primary amine oxidase activity","mitochondrial membrane"],["1.1e-08","2.4e-08","2.4e-08","2.4e-08","3.1e-05","5.4e-05","5.5e-05","5.9e-05","1.2e-04","1.3e-04","2.6e-04","2.7e-04","2.7e-04","5.6e-04","5.6e-04","7.5e-04","1.3e-03","1.4e-03","3.0e-03","3.5e-03","3.5e-03","6.4e-03","7.1e-03","9.1e-03","1.5e-02","1.7e-02","2.1e-02","2.5e-02","2.5e-02","2.6e-02","2.7e-02","2.9e-02","3.4e-02","3.5e-02","3.5e-02","3.9e-02","4.5e-02","4.6e-02","4.8e-02"],[2178,2263,2265,2265,4058,4564,4651,13201,3595,11951,3644,723,723,395,395,818,450,14027,12046,11,1678,1995,20,3,6,475,2165,1244,1244,53,115,14808,770,37,1,81,5,6,750],[21,21,21,21,23,24,46,57,21,82,33,9,9,7,7,9,7,57,51,3,16,24,3,2,2,9,18,11,11,3,3,90,10,2,1,2,2,2,10]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="reduced-terms-cd73" class="section level4">
<h4>Reduced Terms CD73</h4>
<pre class="r"><code>simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)</code></pre>
<p><img src="figure/both_cell_line.Rmd/unnamed-chunk-2-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
<div id="wt" class="section level4">
<h4>WT</h4>
<pre class="r"><code>Gname &lt;- results_CAR %&gt;% filter(significant &amp; get(paste0(test_CAR, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

Uni &lt;- table_unique_CAR %&gt;% filter(unique_OVCAR_WT) %&gt;% pull(name)

#bg_list &lt;- results_CAR %&gt;% pull(name) %&gt;% unique()

GO_name &lt;- c(Gname, Uni)

GO &lt;- gost(query = GO_name, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T)

gostplot(GO, capped = TRUE, interactive = TRUE)</code></pre>
<div id="htmlwidget-11e4c9fd271d7e55a8a6" style="width:768px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-11e4c9fd271d7e55a8a6">{"x":{"data":[{"x":[99.9141024506509,148.956250218286,137.054710538839,120.4393807175,160.565129085943,86.6155951629282,90.7947915552847,149.030390957273,126.401856989695,172.384723738088,172.392528026403,188.57862199045,149.038195245587,160.092969642922,176.571724418759,137.288839188271,121.340776017812,161.482132962884,86.213674314737,124.89172720086,176.579528707074,124.887825056703,120.177937058968,147.953399169887,102.341236116427,85.8858942055326,123.330869537982,83.4041305215562,187.240186544532],"y":[4.87894199799311,4.48136164951742,4.48136164951742,3.14145647775671,2.92066113362715,2.89013650554383,2.52964150868989,2.42473143795797,2.40620508816666,2.36326402347946,2.36326402347946,2.28747982622424,2.14845943122336,2.14129747005977,2.13848889187277,2.08010559207649,2.02314728023933,1.89110548050814,1.87596740915452,1.7304614487896,1.63209298329342,1.59148779807126,1.56217434343517,1.54209518241382,1.50415628887649,1.45974657543371,1.4048134324096,1.31169847413273,1.31031012776329],"text":["GO:0018916 (4) <br> nitrobenzene metabolic process <br> 1.321e-05","GO:0070458 (5) <br> cellular detoxification of nitrogen compound <br> 3.301e-05","GO:0051410 (5) <br> detoxification of nitrogen compound <br> 3.301e-05","GO:0042178 (12) <br> xenobiotic catabolic process <br> 7.220e-04","GO:0097435 (863) <br> supramolecular fiber organization <br> 1.200e-03","GO:0007010 (1473) <br> cytoskeleton organization <br> 1.288e-03","GO:0009636 (254) <br> response to toxic substance <br> 2.954e-03","GO:0070486 (13) <br> leukocyte aggregation <br> 3.761e-03","GO:0045055 (790) <br> regulated exocytosis <br> 3.925e-03","GO:1901685 (21) <br> glutathione derivative metabolic process <br> 4.332e-03","GO:1901687 (21) <br> glutathione derivative biosynthetic process <br> 4.332e-03","GO:1990748 (113) <br> cellular detoxification <br> 5.158e-03","GO:0070488 (2) <br> neutrophil aggregation <br> 7.105e-03","GO:0097237 (121) <br> cellular response to toxic substance <br> 7.223e-03","GO:1902903 (391) <br> regulation of supramolecular fiber organization <br> 7.270e-03","GO:0051493 (535) <br> regulation of cytoskeleton organization <br> 8.316e-03","GO:0042537 (27) <br> benzene-containing compound metabolic process <br> 9.481e-03","GO:0098754 (143) <br> detoxification <br> 1.285e-02","GO:0006887 (906) <br> exocytosis <br> 1.331e-02","GO:0044283 (675) <br> small molecule biosynthetic process <br> 1.860e-02","GO:1902905 (215) <br> positive regulation of supramolecular fiber organization <br> 2.333e-02","GO:0044282 (437) <br> small molecule catabolic process <br> 2.562e-02","GO:0042060 (555) <br> wound healing <br> 2.740e-02","GO:0065009 (3191) <br> regulation of molecular function <br> 2.870e-02","GO:0019752 (1033) <br> carboxylic acid metabolic process <br> 3.132e-02","GO:0006790 (388) <br> sulfur compound metabolic process <br> 3.469e-02","GO:0043436 (1066) <br> oxoacid metabolic process <br> 3.937e-02","GO:0006082 (1098) <br> organic acid metabolic process <br> 4.879e-02","GO:1905907 (12) <br> negative regulation of amyloid fibril formation <br> 4.894e-02"],"key":["GO:0018916","GO:0070458","GO:0051410","GO:0042178","GO:0097435","GO:0007010","GO:0009636","GO:0070486","GO:0045055","GO:1901685","GO:1901687","GO:1990748","GO:0070488","GO:0097237","GO:1902903","GO:0051493","GO:0042537","GO:0098754","GO:0006887","GO:0044283","GO:1902905","GO:0044282","GO:0042060","GO:0065009","GO:0019752","GO:0006790","GO:0043436","GO:0006082","GO:1905907"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[9.11502471894844,9.91402245464372,9.91402245464372,12.3578430925037,19.5650418017661,20.2458621988079,17.8845121597017,12.5473578799537,19.4496464090039,13.6065768521997,13.6065768521997,16.6514395727192,3.77952755905512,16.7601059913514,18.4991429180246,18.9302681653975,14.1184043066181,17.0217449612193,19.6281701225723,19.2421110280542,17.6397000691062,18.6535017817201,18.9799309992957,21.183223223459,19.7972414698237,18.4883938104321,19.8375053456001,19.8752840693548,12.3578430925037],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDatac6cc692d","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[64.5244030636107,69.6488993741428,62.1358335723345,64.4307336717959,54.9037759459705,54.7593689669227,55.6531310804885,59.4896732535677,55.891207451351,68.5717013682731,58.0338947891135,55.7546062549545,55.3213853178112,63.345729883275,59.7823901029888,57.8114299835534,57.0854921969891],"y":[8.44657630798989,8.11454816044201,8.10690567718127,8.10690567718127,6.7065501236321,5.26074457091203,4.94046614528394,4.93086218993409,2.66577939176444,2.27185195425946,2.21416226945824,2.12946219036054,1.8133235463056,1.57206384920399,1.52338147084508,1.49289978684563,1.35176900159023],"text":["GO:0070062 (2178) <br> extracellular exosome <br> 3.576e-09","GO:1903561 (2263) <br> extracellular vesicle <br> 7.682e-09","GO:0043230 (2265) <br> extracellular organelle <br> 7.818e-09","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 7.818e-09","GO:0005615 (3595) <br> extracellular space <br> 1.965e-07","GO:0005576 (4564) <br> extracellular region <br> 5.486e-06","GO:0005829 (5361) <br> cytosol <br> 1.147e-05","GO:0031982 (4058) <br> vesicle <br> 1.173e-05","GO:0005902 (89) <br> microvillus <br> 2.159e-03","GO:0099503 (1027) <br> secretory vesicle <br> 5.347e-03","GO:0030141 (857) <br> secretory granule <br> 6.107e-03","GO:0005862 (4) <br> muscle thin filament tropomyosin <br> 7.422e-03","GO:0005737 (11951) <br> cytoplasm <br> 1.537e-02","GO:0045171 (76) <br> intercellular bridge <br> 2.679e-02","GO:0032432 (78) <br> actin filament bundle <br> 2.997e-02","GO:0030017 (211) <br> sarcomere <br> 3.214e-02","GO:0015629 (517) <br> actin cytoskeleton <br> 4.449e-02"],"key":["GO:0070062","GO:1903561","GO:0043230","GO:0065010","GO:0005615","GO:0005576","GO:0005829","GO:0031982","GO:0005902","GO:0099503","GO:0030141","GO:0005862","GO:0005737","GO:0045171","GO:0032432","GO:0030017","GO:0015629"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.7265886988686,20.7729150330287,20.7739824997153,20.7739824997153,21.3233173034207,21.6004739150545,21.7849818620342,21.4645503400295,16.2647430320528,19.7897716465247,19.5559632816792,9.11502471894844,22.6771653543307,16.0022741255538,16.0458369440117,17.6118481310321,18.8838106504284],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatac6cc692d","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[23.4199102027904,28.1027107572354,28.5358698085216,28.0988084234401,48.5782561815461,29.8158352934032,9.30126653113881,21.773125341144,30.7680047394737,5.83209178705415,3.45166817187795,37.4800188675115,9.64076957133607,28.4344091298419,26.772014933014,3.42825416910572,27.813938056378,16.5908260608915,5.23893705015778,36.8712547954337,10.1129519605759,36.8673524616383,36.8634501278429,23.3340588592923,28.4305067960466],"y":[5.6122012429888,5.30944593522115,3.89514816444831,3.85072611717062,3.77055835000228,3.5696212523345,3.36283624532544,3.31229328454921,3.01623341181557,2.75578883448185,2.3917356682805,2.31891747258189,2.15052374780845,2.08877704108178,1.98743620523472,1.86978740638712,1.80250410325481,1.61737236546688,1.60333029457011,1.59055067337488,1.50210175236254,1.44494978751208,1.44494978751208,1.3583735666542,1.30311234915871],"text":["GO:0033293 (73) <br> monocarboxylic acid binding <br> 2.442e-06","GO:0042803 (679) <br> protein homodimerization activity <br> 4.904e-06","GO:0043295 (11) <br> glutathione binding <br> 1.273e-04","GO:0042802 (2063) <br> identical protein binding <br> 1.410e-04","GO:1900750 (12) <br> oligopeptide binding <br> 1.696e-04","GO:0044877 (1304) <br> protein-containing complex binding <br> 2.694e-04","GO:0005504 (39) <br> fatty acid binding <br> 4.337e-04","GO:0031406 (177) <br> carboxylic acid binding <br> 4.872e-04","GO:0046983 (1073) <br> protein dimerization activity <br> 9.633e-04","GO:0004364 (25) <br> glutathione transferase activity <br> 1.755e-03","GO:0002060 (3) <br> purine nucleobase binding <br> 4.058e-03","GO:0051015 (213) <br> actin filament binding <br> 4.798e-03","GO:0008092 (1001) <br> cytoskeletal protein binding <br> 7.071e-03","GO:0043168 (2413) <br> anion binding <br> 8.151e-03","GO:0035662 (4) <br> Toll-like receptor 4 binding <br> 1.029e-02","GO:0002054 (5) <br> nucleobase binding <br> 1.350e-02","GO:0042277 (312) <br> peptide binding <br> 1.576e-02","GO:0016765 (59) <br> transferase activity, transferring alkyl or aryl (other than methyl) groups <br> 2.413e-02","GO:0004180 (44) <br> carboxypeptidase activity <br> 2.493e-02","GO:0050544 (6) <br> arachidonic acid binding <br> 2.567e-02","GO:0008289 (774) <br> lipid binding <br> 3.147e-02","GO:0050543 (7) <br> icosatetraenoic acid binding <br> 3.590e-02","GO:0050542 (7) <br> icosanoid binding <br> 3.590e-02","GO:0033218 (387) <br> amide binding <br> 4.382e-02","GO:0043167 (6056) <br> ion binding <br> 4.976e-02"],"key":["GO:0033293","GO:0042803","GO:0043295","GO:0042802","GO:1900750","GO:0044877","GO:0005504","GO:0031406","GO:0046983","GO:0004364","GO:0002060","GO:0051015","GO:0008092","GO:0043168","GO:0035662","GO:0002054","GO:0042277","GO:0016765","GO:0004180","GO:0050544","GO:0008289","GO:0050543","GO:0050542","GO:0033218","GO:0043167"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[15.9344231172253,19.2499556985536,12.1469610512846,20.6607303776302,12.3578430925037,20.0931782437329,14.8246539043825,17.3484904766103,19.84587310411,13.9643979652623,7.86026738229134,17.6258464840523,19.7568484243044,20.8502939251815,9.11502471894844,9.91402245464372,18.1808263134707,15.5692174281658,15.0466923813759,10.4966673855122,19.4228101199733,10.9524579477957,10.9524579477957,18.4847905399574,21.9234734329314],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDatac6cc692d","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[198.123588502378],"y":[1.42174696191447],"text":"KEGG:00480 (56) <br> Glutathione metabolism <br> 3.787e-02","key":["KEGG:00480"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(221,68,119,1)","opacity":0.8,"size":15.4779684889999,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(221,68,119,1)"}},"hoveron":"points","set":"SharedDatac6cc692d","name":"KEGG","legendgroup":"KEGG","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[2,49.9206590071537],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8226750487752,71.2217645783655],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.123780619987,193.971385215695],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.873401257316,200.003902016042],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":45.251350032552,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9603295035768,62.5222198135703,134.547582917841,198.938651636679],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.00905891102808911,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"1ed4eb37fa4":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"1ed45e1b8e":{"x":{},"xend":{},"y":{},"yend":{}},"1ed452a03b8b":{"x":{},"xend":{},"y":{},"yend":{}},"1ed4c99a1f":{"x":{},"xend":{},"y":{},"yend":{}},"1ed42e5c6cba":{"x":{},"xend":{},"y":{},"yend":{}},"1ed46ef3957":{"x":{},"y":{}},"1ed4671f77f8":{"yintercept":{}}},"cur_data":"1ed4eb37fa4","visdat":{"1ed4eb37fa4":["function (y) ","x"],"1ed45e1b8e":["function (y) ","x"],"1ed452a03b8b":["function (y) ","x"],"1ed4c99a1f":["function (y) ","x"],"1ed42e5c6cba":["function (y) ","x"],"1ed46ef3957":["function (y) ","x"],"1ed4671f77f8":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatac6cc692d"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-wt" class="section level4">
<h4>table WT</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>GO$result %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
  arrange(p_value) %&gt;%  
  mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))</code></pre>
<div id="htmlwidget-5f2d189442bf377405cf" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-5f2d189442bf377405cf">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:MF","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:MF","KEGG","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:MF"],[2743,4056,2131,2719,278,5490,6690,241,470,1453,6354,18922,15872,6801,6689,11937,7129,1872,5068,11614,7373,21897,2946,983,531,4017,18941,13142,373,24926,24928,9093,29076,3780,1080,1959,18943,21776,25999,496,6775,15932,11845,6349,22132,2843,367,385,6616,12755,26001,3740,831,12754,8937,2441,11547,18665,1528,6976,2080,1023,2759,8936,8935,65,12355,5468,837,2123,28733,6774],["extracellular exosome","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","extracellular space","monocarboxylic acid binding","protein homodimerization activity","extracellular region","cytosol","vesicle","nitrobenzene metabolic process","cellular detoxification of nitrogen compound","detoxification of nitrogen compound","glutathione binding","identical protein binding","oligopeptide binding","protein-containing complex binding","fatty acid binding","carboxylic acid binding","xenobiotic catabolic process","protein dimerization activity","supramolecular fiber organization","cytoskeleton organization","glutathione transferase activity","microvillus","response to toxic substance","leukocyte aggregation","regulated exocytosis","purine nucleobase binding","glutathione derivative metabolic process","glutathione derivative biosynthetic process","actin filament binding","cellular detoxification","secretory vesicle","secretory granule","cytoskeletal protein binding","neutrophil aggregation","cellular response to toxic substance","regulation of supramolecular fiber organization","muscle thin filament tropomyosin","anion binding","regulation of cytoskeleton organization","benzene-containing compound metabolic process","Toll-like receptor 4 binding","detoxification","exocytosis","nucleobase binding","cytoplasm","peptide binding","small molecule biosynthetic process","positive regulation of supramolecular fiber organization","transferase activity, transferring alkyl or aryl (other than methyl) groups","carboxypeptidase activity","small molecule catabolic process","arachidonic acid binding","intercellular bridge","wound healing","regulation of molecular function","actin filament bundle","carboxylic acid metabolic process","lipid binding","sarcomere","sulfur compound metabolic process","icosatetraenoic acid binding","icosanoid binding","Glutathione metabolism","oxoacid metabolic process","amide binding","actin cytoskeleton","organic acid metabolic process","negative regulation of amyloid fibril formation","ion binding"],["3.6e-09","7.7e-09","7.8e-09","7.8e-09","2.0e-07","2.4e-06","4.9e-06","5.5e-06","1.1e-05","1.2e-05","1.3e-05","3.3e-05","3.3e-05","1.3e-04","1.4e-04","1.7e-04","2.7e-04","4.3e-04","4.9e-04","7.2e-04","9.6e-04","1.2e-03","1.3e-03","1.8e-03","2.2e-03","3.0e-03","3.8e-03","3.9e-03","4.1e-03","4.3e-03","4.3e-03","4.8e-03","5.2e-03","5.3e-03","6.1e-03","7.1e-03","7.1e-03","7.2e-03","7.3e-03","7.4e-03","8.2e-03","8.3e-03","9.5e-03","1.0e-02","1.3e-02","1.3e-02","1.3e-02","1.5e-02","1.6e-02","1.9e-02","2.3e-02","2.4e-02","2.5e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.9e-02","3.0e-02","3.1e-02","3.1e-02","3.2e-02","3.5e-02","3.6e-02","3.6e-02","3.8e-02","3.9e-02","4.4e-02","4.4e-02","4.9e-02","4.9e-02","5.0e-02"],[2178,2263,2265,2265,3595,73,679,4564,5361,4058,4,5,5,11,2063,12,1304,39,177,12,1073,863,1473,25,89,254,13,790,3,21,21,213,113,1027,857,1001,2,121,391,4,2413,535,27,4,143,906,5,11951,312,675,215,59,44,437,6,76,555,3191,78,1033,774,211,388,7,7,56,1066,387,517,1098,12,6056],[22,22,22,22,25,6,13,26,30,23,3,3,3,3,19,3,17,4,6,3,13,12,17,3,5,8,3,10,2,3,3,6,5,11,9,11,2,5,8,2,15,9,3,2,6,10,2,38,5,8,6,3,3,7,2,3,10,25,4,8,9,5,6,2,2,3,8,5,7,8,2,23]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="reduced-terms-wt" class="section level4">
<h4>Reduced Terms WT</h4>
<pre class="r"><code>simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)</code></pre>
<p><img src="figure/both_cell_line.Rmd/unnamed-chunk-3-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="ov90-4" class="section level3 tabset tabset-fade">
<h3>OV90</h3>
<div id="cd73-1" class="section level4">
<h4>CD73</h4>
<pre class="r"><code>Gname &lt;- results_90 %&gt;% filter(significant &amp; get(paste0(test_90, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

Uni &lt;- table_unique_90 %&gt;% filter(unique_OV90_CD73) %&gt;% pull(name)

#bg_list &lt;- results_90 %&gt;% pull(name) %&gt;% unique()

GO_name &lt;- c(Gname, Uni)

GO &lt;- gost(query = GO_name, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T)

gostplot(GO, capped = TRUE, interactive = TRUE)</code></pre>
<div id="htmlwidget-d4e0e6de104c38825df0" style="width:768px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-d4e0e6de104c38825df0">{"x":{"data":[{"x":[57.0113372618024,58.3734463344419,58.5685909007227,56.0707404523293,58.5607851180714,67.9745589954541],"y":[2.65267114648809,1.94202006078509,1.93063904960224,1.40503474501146,1.32585349896411,1.30278429413885],"text":["GO:0012506 (1183) <br> vesicle membrane <br> 2.225e-03","GO:0030904 (21) <br> retromer complex <br> 1.143e-02","GO:0031090 (3644) <br> organelle membrane <br> 1.173e-02","GO:0005955 (5) <br> calcineurin complex <br> 3.935e-02","GO:0031088 (6) <br> platelet dense granule membrane <br> 4.722e-02","GO:0098796 (1326) <br> membrane protein complex <br> 4.980e-02"],"key":["GO:0012506","GO:0030904","GO:0031090","GO:0005955","GO:0031088","GO:0098796"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[21.333138581955,15.2938400962575,22.6771653543307,12.151263812459,12.6127319184972,21.47411814965],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatab9ae2f4d","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[42.662318147764,8.68469779147022],"y":[1.86241822912853,1.38529697440917],"text":["GO:0098695 (1) <br> inositol 1,4,5-trisphosphate receptor activity involved in regulation of postsynaptic cytosolic calcium levels <br> 1.373e-02","GO:0005220 (3) <br> inositol 1,4,5-trisphosphate-sensitive calcium-release channel activity <br> 4.118e-02"],"key":["GO:0098695","GO:0005220"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[3.77952755905512,10.6962548077426],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDatab9ae2f4d","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,49.9206590071537],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8226750487752,71.2217645783655],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.123780619987,193.971385215695],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.873401257316,200.003902016042],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":45.251350032552,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9603295035768,62.5222198135703,134.547582917841,198.938651636679],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.00905891102808911,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"1ed45dcc6da9":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"1ed4a73360e":{"x":{},"xend":{},"y":{},"yend":{}},"1ed466e43031":{"x":{},"xend":{},"y":{},"yend":{}},"1ed4565958fa":{"x":{},"xend":{},"y":{},"yend":{}},"1ed417d60ca":{"x":{},"xend":{},"y":{},"yend":{}},"1ed47e7f4844":{"x":{},"y":{}},"1ed41ee76c89":{"yintercept":{}}},"cur_data":"1ed45dcc6da9","visdat":{"1ed45dcc6da9":["function (y) ","x"],"1ed4a73360e":["function (y) ","x"],"1ed466e43031":["function (y) ","x"],"1ed4565958fa":["function (y) ","x"],"1ed417d60ca":["function (y) ","x"],"1ed47e7f4844":["function (y) ","x"],"1ed41ee76c89":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatab9ae2f4d"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>GO$result %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
  arrange(p_value) %&gt;%  
  mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))</code></pre>
<div id="htmlwidget-05e2a89b98a2d9619f05" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-05e2a89b98a2d9619f05">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8"],["GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:MF","GO:CC","GO:CC"],[818,1167,1217,10421,577,1714,1215,3627],["vesicle membrane","retromer complex","organelle membrane","inositol 1,4,5-trisphosphate receptor activity involved in regulation of postsynaptic cytosolic calcium levels","calcineurin complex","inositol 1,4,5-trisphosphate-sensitive calcium-release channel activity","platelet dense granule membrane","membrane protein complex"],["2.2e-03","1.1e-02","1.2e-02","1.4e-02","3.9e-02","4.1e-02","4.7e-02","5.0e-02"],[1183,21,3644,1,5,3,6,1326],[7,2,10,1,1,1,1,6]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="wt-1" class="section level4">
<h4>WT</h4>
<pre class="r"><code>Gname &lt;- results_90 %&gt;% filter(significant &amp; get(paste0(test_90, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

Uni &lt;- table_unique_90 %&gt;% filter(unique_OV90_WT) %&gt;% pull(name)

#bg_list &lt;- results_90 %&gt;% pull(name) %&gt;% unique()

GO_name &lt;- c(Gname, Uni)

GO &lt;- gost(query = GO_name, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T)

gostplot(GO, capped = TRUE, interactive = TRUE)</code></pre>
<div id="htmlwidget-bad0dbf07cb4b9aadd26" style="width:768px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-bad0dbf07cb4b9aadd26">{"x":{"data":[{"x":[58.2095248987661,67.7989288858014],"y":[2.45676315451931,1.45219624335495],"text":["GO:0030667 (308) <br> secretory granule membrane <br> 3.493e-03","GO:0098588 (2165) <br> bounding membrane of organelle <br> 3.530e-02"],"key":["GO:0030667","GO:0098588"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[12.1261036126974,22.6771653543307],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData7b5320ed","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[9.99588194671481],"y":[1.49686333709525],"text":"GO:0008237 (192) <br> metallopeptidase activity <br> 3.185e-02","key":["GO:0008237"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":3.77952755905512,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData7b5320ed","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[2,49.9206590071537],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8226750487752,71.2217645783655],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.123780619987,193.971385215695],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.873401257316,200.003902016042],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":45.251350032552,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9603295035768,62.5222198135703,134.547582917841,198.938651636679],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.00905891102808911,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"1ed4362f491e":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"1ed436d14046":{"x":{},"xend":{},"y":{},"yend":{}},"1ed446d03aad":{"x":{},"xend":{},"y":{},"yend":{}},"1ed47d530b3":{"x":{},"xend":{},"y":{},"yend":{}},"1ed430944fe":{"x":{},"xend":{},"y":{},"yend":{}},"1ed456b670c3":{"x":{},"y":{}},"1ed45de76367":{"yintercept":{}}},"cur_data":"1ed4362f491e","visdat":{"1ed4362f491e":["function (y) ","x"],"1ed436d14046":["function (y) ","x"],"1ed446d03aad":["function (y) ","x"],"1ed47d530b3":["function (y) ","x"],"1ed430944fe":["function (y) ","x"],"1ed456b670c3":["function (y) ","x"],"1ed45de76367":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData7b5320ed"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>GO$result %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
  arrange(p_value) %&gt;%  
  mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))</code></pre>
<div id="htmlwidget-57b0d00a687b8ee39892" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-57b0d00a687b8ee39892">{"x":{"filter":"none","data":[["1","2","3"],["GO:CC","GO:MF","GO:CC"],[1125,2050,3582],["secretory granule membrane","metallopeptidase activity","bounding membrane of organelle"],["3.5e-03","3.2e-02","3.5e-02"],[308,192,2165],[3,2,4]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>header_style &lt;- createStyle(
  fontSize = 12,
  fontColour = &quot;#0f0f0f&quot;,
  fgFill = &quot;#faf2ca&quot;,
  halign = &quot;center&quot;,
  border = &quot;TopBottomLeftRight&quot;
)

body_style &lt;- createStyle(
  halign = &quot;center&quot;,
  border = &quot;TopBottomLeftRight&quot;
)

excel &lt;- createWorkbook()

## sheet1

addWorksheet(excel, sheetName = &quot;OVCAR_CD73_vs_WT&quot;, gridLines = F)

writeDataTable(excel, sheet = &quot;OVCAR_CD73_vs_WT&quot;, x = results_DEP_OVCAR, keepNA = T, na.string = &quot;NaN&quot;)

n_row &lt;- results_DEP_OVCAR %&gt;% nrow() + 1

n_col &lt;- results_DEP_OVCAR %&gt;% ncol()

setColWidths(excel, sheet = &quot;OVCAR_CD73_vs_WT&quot;, cols = 1:n_col, widths = 21)

addStyle(excel, sheet = &quot;OVCAR_CD73_vs_WT&quot;, style = header_style, rows = 1, cols = 1:n_col, gridExpand = T)

addStyle(excel, sheet = &quot;OVCAR_CD73_vs_WT&quot;, style = body_style, rows = 2:n_row, cols = 1:n_col, gridExpand = T)

## sheet2

addWorksheet(excel, sheetName = &quot;Unique_OVCAR&quot;, gridLines = F)

writeDataTable(excel, sheet = &quot;Unique_OVCAR&quot;, x = result_unique_OVCAR, keepNA = T, na.string = &quot;NaN&quot;)

n_row &lt;- result_unique_OVCAR %&gt;% nrow() + 1

n_col &lt;- result_unique_OVCAR %&gt;% ncol()

setColWidths(excel, sheet = &quot;Unique_OVCAR&quot;, cols = 1:n_col, widths = 21)

addStyle(excel, sheet = &quot;Unique_OVCAR&quot;, style = header_style, rows = 1, cols = 1:n_col, gridExpand = T)

addStyle(excel, sheet = &quot;Unique_OVCAR&quot;, style = body_style, rows = 2:n_row, cols = 1:n_col, gridExpand = T)

## sheet3

addWorksheet(excel, sheetName = &quot;OV90_CD73_vs_WT&quot;, gridLines = F)

writeDataTable(excel, sheet = &quot;OV90_CD73_vs_WT&quot;, x = results_DEP_OV90, keepNA = T, na.string = &quot;NaN&quot;)

n_row &lt;- results_DEP_OV90 %&gt;% nrow() + 1

n_col &lt;- results_DEP_OV90 %&gt;% ncol()

setColWidths(excel, sheet = &quot;OV90_CD73_vs_WT&quot;, cols = 1:n_col, widths = 21)

addStyle(excel, sheet = &quot;OV90_CD73_vs_WT&quot;, style = header_style, rows = 1, cols = 1:n_col, gridExpand = T)

addStyle(excel, sheet = &quot;OV90_CD73_vs_WT&quot;, style = body_style, rows = 2:n_row, cols = 1:n_col, gridExpand = T)

## sheet4

addWorksheet(excel, sheetName = &quot;Unique_OV90&quot;, gridLines = F)

writeDataTable(excel, sheet = &quot;Unique_OV90&quot;, x = result_unique_OV90, keepNA = T, na.string = &quot;NaN&quot;)

n_row &lt;- result_unique_OV90 %&gt;% nrow() + 1

n_col &lt;- result_unique_OV90 %&gt;% ncol()

setColWidths(excel, sheet = &quot;Unique_OV90&quot;, cols = 1:n_col, widths = 21)

addStyle(excel, sheet = &quot;Unique_OV90&quot;, style = header_style, rows = 1, cols = 1:n_col, gridExpand = T)

addStyle(excel, sheet = &quot;Unique_OV90&quot;, style = body_style, rows = 2:n_row, cols = 1:n_col, gridExpand = T)

## save

saveWorkbook(excel, here::here(&quot;output/results_UC_CD73_210716.xlsx&quot;), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
 [3] AnnotationDbi_1.52.0        openxlsx_4.2.4             
 [5] gprofiler2_0.2.0            naniar_0.6.1               
 [7] visdat_0.5.3                ggrepel_0.9.1              
 [9] UpSetR_1.4.0                ggstatsplot_0.8.0          
[11] datapasta_3.1.0             here_1.0.1                 
[13] patchwork_1.1.1             stringr_1.4.0              
[15] DT_0.18                     enrichR_3.0                
[17] ggplot2_3.3.5               tibble_3.1.2               
[19] preprocessCore_1.52.1       SummarizedExperiment_1.20.0
[21] Biobase_2.50.0              GenomicRanges_1.42.0       
[23] GenomeInfoDb_1.26.2         IRanges_2.24.1             
[25] S4Vectors_0.28.1            BiocGenerics_0.36.0        
[27] MatrixGenerics_1.2.1        matrixStats_0.59.0         
[29] DEP_1.12.0                  tidyr_1.1.3                
[31] dplyr_1.0.7                 MBQN_2.2.0                 
[33] workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] utf8_1.2.1                shinydashboard_0.7.1     
  [3] gmm_1.6-6                 tidyselect_1.1.1         
  [5] RSQLite_2.2.7             htmlwidgets_1.5.3        
  [7] grid_4.0.3                BiocParallel_1.24.1      
  [9] gmp_0.6-2                 norm_1.0-9.5             
 [11] munsell_0.5.0             codetools_0.2-16         
 [13] effectsize_0.4.5          withr_2.4.2              
 [15] GOSemSim_2.16.1           colorspace_2.0-2         
 [17] highr_0.9                 NLP_0.2-1                
 [19] knitr_1.33                ipmisc_6.0.2             
 [21] ggsignif_0.6.2            mzID_1.28.0              
 [23] labeling_0.4.2            git2r_0.28.0             
 [25] slam_0.1-48               GenomeInfoDbData_1.2.4   
 [27] farver_2.1.0              pheatmap_1.0.12          
 [29] bit64_4.0.5               rprojroot_2.0.2          
 [31] coda_0.19-4               vctrs_0.3.8              
 [33] generics_0.1.0            xfun_0.24                
 [35] BiocFileCache_1.14.0      BWStest_0.2.2            
 [37] R6_2.5.0                  doParallel_1.0.16        
 [39] clue_0.3-59               BayesFactor_0.9.12-4.2   
 [41] bitops_1.0-7              cachem_1.0.5             
 [43] reshape_0.8.8             DelayedArray_0.16.2      
 [45] assertthat_0.2.1          promises_1.2.0.1         
 [47] scales_1.1.1              gtable_0.3.0             
 [49] Cairo_1.5-12.2            affy_1.68.0              
 [51] multcompView_0.1-8        sandwich_3.0-1           
 [53] rlang_0.4.11              MatrixModels_0.5-0       
 [55] zeallot_0.1.0             mzR_2.24.1               
 [57] GlobalOptions_0.1.2       PMCMRplus_1.9.0          
 [59] lazyeval_0.2.2            impute_1.64.0            
 [61] wordcloud_2.6             BiocManager_1.30.16      
 [63] yaml_2.2.1                crosstalk_1.1.1          
 [65] httpuv_1.6.1              tools_4.0.3              
 [67] gridBase_0.4-7            affyio_1.60.0            
 [69] ellipsis_0.3.2            jquerylib_0.1.4          
 [71] RColorBrewer_1.1-2        WRS2_1.1-2               
 [73] MSnbase_2.15.7            Rcpp_1.0.6               
 [75] plyr_1.8.6                zlibbioc_1.36.0          
 [77] purrr_0.3.4               RCurl_1.98-1.3           
 [79] pbapply_1.4-3             GetoptLong_1.0.5         
 [81] correlation_0.6.1         zoo_1.8-9                
 [83] cluster_2.1.0             fs_1.5.0                 
 [85] magrittr_2.0.1            data.table_1.14.0        
 [87] circlize_0.4.13           pcaMethods_1.82.0        
 [89] mvtnorm_1.1-2             whisker_0.4              
 [91] ProtGenerics_1.22.0       hms_1.1.0                
 [93] mime_0.11                 evaluate_0.14            
 [95] xtable_1.8-4              XML_3.99-0.6             
 [97] pairwiseComparisons_3.1.6 gridExtra_2.3            
 [99] shape_1.4.6               compiler_4.0.3           
[101] ncdf4_1.17                crayon_1.4.1             
[103] htmltools_0.5.1.1         mc2d_0.1-21              
[105] later_1.2.0               DBI_1.1.1                
[107] SuppDists_1.1-9.5         kSamples_1.2-9           
[109] dbplyr_2.1.1              ComplexHeatmap_2.6.2     
[111] MASS_7.3-53               tmvtnorm_1.4-10          
[113] rappdirs_0.3.3            Matrix_1.2-18            
[115] readr_1.4.0               vsn_3.58.0               
[117] imputeLCMD_2.0            igraph_1.2.6             
[119] insight_0.14.2            pkgconfig_2.0.3          
[121] statsExpressions_1.1.0    plotly_4.9.4.1           
[123] xml2_1.3.2                MALDIquant_1.19.3        
[125] paletteer_1.3.0           foreach_1.5.1            
[127] bslib_0.2.5.1             XVector_0.30.0           
[129] digest_0.6.27             parameters_0.14.0        
[131] tm_0.7-8                  rmarkdown_2.9            
[133] curl_4.3.2                shiny_1.6.0              
[135] gtools_3.9.2              rjson_0.2.20             
[137] lifecycle_1.0.0           jsonlite_1.7.2           
[139] viridisLite_0.4.0         limma_3.46.0             
[141] fansi_0.5.0               pillar_1.6.1             
[143] lattice_0.20-41           treemap_2.4-2            
[145] GO.db_3.12.1              fastmap_1.1.0            
[147] httr_1.4.2                glue_1.4.2               
[149] zip_2.2.0                 bayestestR_0.10.0        
[151] png_0.1-7                 iterators_1.0.13         
[153] bit_4.0.4                 stringi_1.6.2            
[155] sass_0.4.0                performance_0.7.2        
[157] rematch2_2.1.2            blob_1.2.1               
[159] memoise_2.0.0             Rmpfr_0.8-4              </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
